pages.es/common/cpdf.md

```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> Interfaz de línea de comandos para manipular documentos PDF existentes de diferentes maneras.
+> Más información: <https://www.coherentpdf.com/cpdfmanudel/cpdfmanudel.html>.
+
+- Selecciona las páginas 1, 2, 3 y 6 del documento fuente y escribirlas en el documento objetivo:
+
+`cpdf {{ruta/del/documento_fuente.pdf}} 1-3,6 -o {{ruta/del/documento_objetivo.pdf}}`
```

Comment:
1-3, 6 should be in tokens.

Reply:
```suggestion
`cpdf {{ruta/del/documento_fuente.pdf}} {{1-3,6}} -o {{ruta/del/documento_objetivo.pdf}}`
```

Reply:
Fixed @zdroid, @sbrl!

**P.S.** Happy new year!

Reply:
Feliz año nuevo! :fireworks: